### PR TITLE
Separate Marketplace apis out

### PR DESCRIPTION
**Pull request by kristinfritsch**
Separate marketplace apis into separate api viewer.

Screen Shot 2020-03-16 at 14 17 29

### CODE REVIEW

File: components/MarketplaceInfoFields.vue [hunk 1]
```diff
@@ -1,19 +1,24 @@
 <template>
   <div>
     <v-text-field
-      v-model="walletRefId"
-      :value="value.walletRefId"
-      label="Wallet RefId"
+      v-model="marketplaceInfo.enduserWallet"
```
Comment (jcantwell) [inline components/MarketplaceInfoFields.vue]: do we want to use `walletAccountNumber` for now until we complete https://circlepay.atlassian.net/browse/BRAAV-1022?

File: layouts/default.vue [hunk 2]
```diff
@@ -15,18 +15,15 @@
             </v-list-item-title>
           </v-list-item-content>
         </v-list-item>
-        <v-list-item>
```
Comment (kristinfritsch) [inline layouts/default.vue]: Updates the navigation to use `v-list-group` and adds a different nav for marketplace api key

File: layouts/error.vue [hunk 3]
```diff
@@ -1,42 +1,29 @@
 <template>
   <v-app dark>
     <h1 v-if="error.statusCode === 404">
-      {{ pageNotFound }}
```
Comment (kristinfritsch) [inline layouts/error.vue]: Update error page to show error if wrong route is accessed using a merchant vs marketplace api key

File: lib/marketplaceApi.ts [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+ * @param {String} pageSize
+ * @param {String} merchantId
+ */
+function getPayments(
+  pageBefore: string,
+  pageAfter: string,
+  pageSize: string,
+  merchantId: string
+) {
+  const nullIfEmpty = (prop: string) => {
```
Comment (kristinfritsch) [inline lib/marketplaceApi.ts]: Axios omits query params if they are null, not if there are empty strings

File: layouts/default.vue [hunk 5]
```diff
@@ -128,7 +145,49 @@ export default class DefaultLayoutsClass extends Vue {
...
+    {
+      title: 'GET /payments/{id}',
+      to: '/debug/marketplace/payments/details'
+    },
+    {
+      title: 'POST /payments',
+      to: '/debug/marketplace/payments/create'
+    },
+    {
+      title: 'POST /payments/{id}/cancel',
```
Comment (jcantwell) [inline layouts/default.vue]: minor: does it make sense to update the title properties as well?

Comment (kristinfritsch) [reply layouts/default.vue]: yes, great catch

**Pull request merged**